### DISCUSSION TITLE
添加一个”神奇代码岛访问入口“

### DISCUSSION THREAD

**Discussion by DT0O1** [Meloong-Git/PCL#374] [提问] [answered]
神奇代码岛是一个多人联机3D创作平台，即点即玩免下载！
（注：需要一定JS编程基础！）
这里比MC还好Van♂，快来吧！创作出你最棒的作品！
链接：[https://box3.codemao.cn/](url)

**Comment by WForst-Breeze** [comment]
草，我依稀记得PCL2是个**MC启动器**🤔

**Comment by Pigeon0v0** [comment, marked as answer]
PCL2 是 Minecraft 启动器...

提交 Discussion 时过过脑子

**Reply by SuoHouDaiShi** [reply]
正确的，中肯的，一针见血的

**Comment by StudyingPy** [comment]
有没有可能它就是来打广告的……

**Reply by Xiaolv114518** [reply]
编程猫的学员很多都这样的……
绝大部分都是”缺爱“的xxs，他们希望自己支持的东西被认可

**Comment by Xiaolv114518** [comment]
PCL是mc启动器，不是bcm社区入口
想要这个功能可以自己写个主页，不会写可以直接基于教学文件进行修改
但请不要给bcm招黑，谢谢

**Comment by SuoHouDaiShi** [comment]
有没有一种可能PCL是**MC启动器而不是百宝箱**
另外：`DON'T ASK QUESTIONS THE STUPID WAYS`
提交反馈/讨论时请**先动动脑子**，脑子不要**可以捐了**

**Reply by so1ve** [reply]
好了没必要挖坟…

**Reply by iPodtouch6557** [reply]
**再不行就lock得了**

**Reply by WForst-Breeze** [reply]
友善态度讨论，尽可能避免挖坟，谢谢——
